src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderContactPersonRepository.kt

```diff
@@ -1,8 +1,8 @@
 package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository
 
-import org.springframework.data.repository.CrudRepository
+import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.OffenderContactPerson
 
 @Repository
-interface OffenderContactPersonRepository : CrudRepository<OffenderContactPerson, Long>
+interface OffenderContactPersonRepository : JpaRepository<OffenderContactPerson, Long>
```

Comment:
needs to a JPA for the saveAndFlush that is needed since we updated created dates straight after a save when testing